WalletWasabi/Blockchain/Transactions/TransactionFactory.cs

```diff
@@ -65,15 +65,16 @@ public TransactionFactory(Network network, KeyManager keyManager, ICoinsView coi
 		}
 
 		// Get allowed coins to spend.
-		var availableCoinsView = Coins.Unspent();
+		var availableCoinsView = Coins.Available().Unspent();
```

Comment:
I don't think `TransactionFactory` should be touched at all. This is something that only needs to be fixed in the Fluent project, or am I missing something?

Reply:
I'm gonna double check this, we might not even need this anymore.